modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,579 @@
+[source,yaml]
+----
+  providerID: baremetalhost:///openshift-machine-api/openshift-master-12/3354bdac-61d8-410f-be5b-6a395b056135
+----
+
+.. Remove the `metadata.annotations` and `metadata.generation` fields:
++
+[source,yaml]
+----
+  annotations:
+    machine.openshift.io/instance-state: externally provisioned
+  ...
+  generation: 2
+----
+
+.. Remove the `spec.conditions`, `spec.lastUpdated`, `spec.nodeRef` and `spec.phase' fields:
++
+[source,yaml]
+----
+  lastTransitionTime: "2022-08-03T08:40:36Z"
+message: 'Drain operation currently blocked by: [{Name:EtcdQuorumOperator Owner:clusteroperator/etcd}]'
+reason: HookPresent
+severity: Warning
+status: "False"
+
+type: Drainable
+lastTransitionTime: "2022-08-03T08:39:55Z"
+status: "True"
+type: InstanceExists
+
+lastTransitionTime: "2022-08-03T08:36:37Z"
+status: "True"
+type: Terminable
+lastUpdated: "2022-08-03T08:40:36Z"
+nodeRef:
+kind: Node
+name: openshift-master-2
+uid: 788df282-6507-4ea2-9a43-24f237ccbc3c
+phase: Running
+----
+
+. Ensure that the Bare Metal Operator is available:
++
+[source,terminal]
+----
+$ oc get clusteroperator baremetal
+----
++
+.Example output
+[source,terminal]
+----
+NAME        VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+baremetal   4.10.12   True        False         False      3d15h
+----
+
+. Delete the machine of the unhealthy member:
++
+[source,terminal]
+----
+$ oc delete machine -n openshift-machine-api clustername-8qw5l-master-2 <1>
+----
++
+If deletion of the machine is delayed for any reason or the command is obstructed and delayed, you can force deletion by removing the machine object finalizer field. 
++
+[IMPORTANT]
+====
+Do not interrupt machine deletion with `control-c`. You must allow the command to proceed to completion. Open a new terminal window to edit and delete the finalizer fields.
+====
++
+[source,terminal]
+----
+$ oc edit machine -n openshift-machine-api clustername-8qw5l-master-0
+----
++
+.. Find and delete the fields:
++
+[source,terminal]
+----
+finalizers:
+- machine.machine.openshift.io
+----
++
+and save the change. 
++
+[source,terminal]
+----
+machine.machine.openshift.io/clustername8qw5l-master-0 edited
+----
++
+.. Verify the machine was deleted:
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api -o wide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            PHASE     TYPE   REGION   ZONE   AGE     NODE                 PROVIDERID                                                                                       STATE
+ocp-edge-4zg89-master-10         Running                          3h11m   openshift-master-0   baremetalhost:///openshift-machine-api/openshift-master-10/da1ebe11-3ff2-41c5-b099-0aa41222964e   externally provisioned
+ocp-edge-4zg89-master-11         Running                          3h11m   openshift-master-1   baremetalhost:///openshift-machine-api/openshift-master-1/d9f9acbc-329c-475e-8d81-03b20280a3e1   externally provisioned
```

Comment:
please, change here as well to 0 and 1

Reply:
Thank you! Those have "tab" instead of "space" so my search missed those.

Reply:
Done! Also found another glitch and corrected it, so thank you, twice!

Reply:
why I still see 10 and 11?